xrpl4j-model/src/main/java/org/xrpl/xrpl4j/model/flags/Flags.java

```diff
@@ -1294,4 +1295,353 @@ public PaymentChannelClaimFlags build() {
+     * If set, indicates that this NfT can be transferred. This flag has no
+     * effect if the token is being transferred from the issuer or to the
+     * issuer.
+     *
+     * @return {@code true} if {@code tfTransferable} is set, otherwise {@code false}.
+     */
+    public boolean tfTransferable() {
+      return this.isSet(TRANSFERABLE);
+    }
+
+    /**
+     * A builder class for {@link NfTokenMintFlags}.
+     */
+    public static class Builder {
+      boolean tfFullyCanonicalSig = true;
+      boolean tfBurnable = false;
+      boolean tfOnlyXRP = false;
+      boolean tfTrustLine = false;
+      boolean tfTransferable = false;
+
+      /**
+       * Set {@code tfFullyCanonicalSig} to the given value.
+       *
+       * @param tfFullyCanonicalSig A boolean value.
+       *
+       * @return The same {@link NfTokenMintFlags.Builder}.
+       */
+      public NfTokenMintFlags.Builder tfFullyCanonicalSig(boolean tfFullyCanonicalSig) {
+        this.tfFullyCanonicalSig = tfFullyCanonicalSig;
+        return this;
+      }
+
+      /**
+       * Set {@code tfBurnable} to the given value.
+       *
+       * @param tfBurnable A boolean value.
+       *
+       * @return The same {@link NfTokenMintFlags.Builder}.
+       */
+      public NfTokenMintFlags.Builder tfBurnable(boolean tfBurnable) {
+        this.tfBurnable = tfBurnable;
+        return this;
+      }
+
+      /**
+       * Set {@code tfOnlyXRP} to the given value.
+       *
+       * @param tfOnlyXRP A boolean value.
+       *
+       * @return The same {@link NfTokenMintFlags.Builder}.
+       */
+      public NfTokenMintFlags.Builder tfOnlyXRP(boolean tfOnlyXRP) {
+        this.tfOnlyXRP = tfOnlyXRP;
+        return this;
+      }
+
+      /**
+       * Set {@code tfTrustLine} to the given value.
+       *
+       * @param tfTrustLine A boolean value.
+       *
+       * @return The same {@link NfTokenMintFlags.Builder}.
+       */
+      public NfTokenMintFlags.Builder tfTrustLine(boolean tfTrustLine) {
+        this.tfTrustLine = tfTrustLine;
+        return this;
+      }
+
+      /**
+       * Set {@code tfTransferable} to the given value.
+       *
+       * @param tfTransferable A boolean value.
+       *
+       * @return The same {@link NfTokenMintFlags.Builder}.
+       */
+      public NfTokenMintFlags.Builder tfTransferable(boolean tfTransferable) {
+        this.tfTransferable = tfTransferable;
+        return this;
+      }
+
+      /**
+       * Build a new {@link NfTokenMintFlags} from the current boolean values.
+       *
+       * @return A new {@link NfTokenMintFlags}.
+       */
+      public NfTokenMintFlags build() {
+        return NfTokenMintFlags.of(tfFullyCanonicalSig, tfBurnable, tfOnlyXRP, tfTrustLine, tfTransferable);
+      }
+    }
+  }
+
+  /**
+   * A set of static {@link Flags} which can be set on {@link org.xrpl.xrpl4j.model.transactions.NfTokenCreateOffer}s.
+   */
+  public static class NfTokenCreateOfferFlags extends Flags {
```

Comment:
Since these flags are for the `NfTokenCreateOffer` transaction, this should extend `TransactionFlags`.